src/OpenColorIO/GPUProcessor.cpp

```diff
@@ -0,0 +1,299 @@
+/*

```

Comment:
General question. Some file/class names use the "GPU" casing, and others use "Gpu". Should we standardise on one?

Reply:
Good catch, the acronyms writing is quite inconsistent. Yes, we will have to update the coding style, and later have clang-tidy running in the CI build.